### PR TITLE
Better help command

### PR DESCRIPTION
**Pull request by lyrixx**
* help the default command
* display also all tasks

image

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -181,11 +185,15 @@ def help(c):
...
                     scheme = 'https' if 'https' in router['using'] else router['using'][0]
-                    print(Fore.YELLOW + "* " + scheme + "://" + host)
+                    print("* " + Fore.YELLOW + scheme + "://" + host + Fore.RESET)
+        print('')
     except:
         pass
 
 
+    print('Run ' + Fore.GREEN + 'inv -l' + Fore.RESET + ' to list all the available tasks.')
+    c.run('inv --list')
```
Comment (pyrech) [inline tasks.py:195]: Wouldn't that list "hide" the urls at the end of `fab start`?

Comment (lyrixx) [reply tasks.py:195]: We could put the list before the URL, WDYT ?

Comment (pyrech) [reply tasks.py:195]: :+1:

**Comment by lyrixx** [comment]
New version 
image

**Pull request merged**